mypy/typeshed/stdlib/typing.pyi

```diff
@@ -676,18 +676,18 @@ class NamedTuple(Tuple[Any, ...]):
     def _replace(self: _T, **kwargs: Any) -> _T: ...
 
 # Internal mypy fallback type for all typed dicts (does not exist at runtime)
-class _TypedDict(Mapping[str, object], metaclass=ABCMeta):
+class _TypedDict(Mapping[str, _VT], metaclass=ABCMeta):
```

Comment:
Typeshed needs to be updated separately. And then synced to mypy, see: https://github.com/python/mypy/blob/master/misc/sync-typeshed.py and https://github.com/python/mypy/blob/master/misc/cherry-pick-typeshed.py

Reply:
Basically, I should update 2 files?

https://github.com/python/typeshed/blob/master/stdlib/typing.pyi
https://github.com/python/typeshed/blob/master/stdlib/typing_extensions.pyi

Reply:
Yes, looks like it! Please, don't forget what other type-checker maintainers think about this change. They might not be ready for it. Since typeshed is used by every other type checker, we need to be sure that this is fine 👍

Reply:
@sobolevn  I have created issue - https://github.com/python/typing/issues/1003